### ISSUE TITLE
Demo.py and python3

### ISSUE THREAD

**Issue by stg7** [mtrenkmann/phrasefinder-client-python#1]
in the first line (magic) there is a space that prevents defined python interpreter from running.

would be nice if demo runs with python3 but this is not the case (wrong print syntax and urllib error)
```
 124, in search
    file = urllibx.urlopen(_to_url(query, options))
AttributeError: module 'urllib' has no attribute 'urlopen'
```

**Comment by mtrenkmann** [comment]
I did no testing for Python3. Could you supply a fix as PR?

**Comment by mtrenkmann** [comment]
I'll fix there error in demo.py myself, because its a minor one.

**Comment by mtrenkmann** [comment]
Fixed by merged PR #2
